UI.tsv

```diff
@@ -519,7 +519,7 @@ UI_20150317_000518	{@st43}Quest{/}
 UI_20150317_000519	${@st59b}Quests{nl}{@st59s} - When you are on a new quest, it will be added to the quest tracker. {nl} - A main quest is displayed in yellow color, while a subquest is displayed in blue. {nl} - The arrow on the character's foot leads to the location of your quest. {nl} - If you completed a quest, a 'Return' icon is displayed on the quest tracker. {nl}    (Teleports you to the Quest NPC using the Backspace key or a mouse-click) {nl} - Add or remove quests in the quest tracker by ticking or unticking the box in front of the quest name. {/}
 UI_20150317_000520	{@st41b}In Progress{/}
 UI_20150317_000521	{@st41b}Abandon{/}
-UI_20150317_000522	{@st43}{s22}Quest Information{/}
+UI_20150317_000522	{@st43}{s22}$Quest Information{/}
```

Comment:
Put the '$' completely in front of the line, so it's easier for IMC to remove it once they start doing this.